Dockerfile

```diff
@@ -2,14 +2,15 @@ FROM golang AS builder
 MAINTAINER "Cuong Manh Le <cuong.manhle.vn@gmail.com>"
 
 RUN apt-get update && \
+    dpkg --add-architecture arm64 &&\
     apt-get install -y --no-install-recommends build-essential && \
     apt-get clean && \
     mkdir -p "$GOPATH/src/github.com/bitnami-labs/kubewatch"
 
 ADD . "$GOPATH/src/github.com/bitnami-labs/kubewatch"
 
 RUN cd "$GOPATH/src/github.com/bitnami-labs/kubewatch" && \
-    CGO_ENABLED=0 GOOS=linux GOARCH=amd64 go build -a --installsuffix cgo --ldflags="-s" -o /kubewatch
+    CGO_ENABLED=0 GOOS=linux GOARCH=$(dpkg --print-architecture) go build -a --installsuffix cgo --ldflags="-s" -o /kubewatch
```

Comment:
This will build images for both amd and arm, right?

Reply:
yes, so the $(dpkg --print-architecture) will print the current architecture (amd/arm) and buildx ensures we run build for both platforms